serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-custom-dashboard-page.adoc

```diff
@@ -0,0 +1,21 @@
+= Dashboards in Kogito Serverless Workflow Tools extension
+
+In Kogito Serverless Workflow Tools, the Dashboards page is used to displays the available dashboard files. The page displays a list of available dashboard and add filters to the list.
+
+.Dashboard page
+image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-dashboard-list.png[]
+
+The table on the Dashboards page displays the following details:
+
+* *Name*: Displays the name of a custom dashboard.
+* *Last Modified*: Displays the time reference indicating when the custom dashboard is last updated.
+
+To filter the dashboard based on name, enter a name in the *Filter by name* field and click *Apply filter* button.
+
+.Example filters applied on the dashboards
+image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-filters-in-dashbaords.png[]
+
+In the dashboard table, the *name* column contains a clickable link, which enables you to navigate to the *Dashboard Details* page.
```

Comment:
I would setup 2 subsections to this page:

- Filtering the dashboard(s)
- Accessing a dashboard details

That way we would have a right menu with the direct link to what we want to achieve. wdyt ?

Reply:
done